Fix Grafana example (#2750)
jaegertracing/jaeger: examples/grafana-integration/README.md

### Explore with Loki

> Currently, the most powerful way to correlate application logs with traces can be performed via Grafana's Explore interface.

Reviewer: please remove the sales pitch:
```suggestion
It is possible to correlate application logs with traces via Grafana's Explore interface.
```

Author: I absolutely agree :smiley:
